core/src/main/java/org/apache/calcite/sql/SqlDropQuark.java

```diff
@@ -27,9 +27,9 @@
  * A <code>SqlDropQuark</code> is a node of a parse tree which represents a DROP DDL
  * statements for Quark.
  */
-public class SqlDropQuark extends SqlCall {
-  public static final SqlSpecialOperator OPERATOR =
-      new SqlSpecialOperator("DROP_QUARK", SqlKind.OTHER_DDL);
+public abstract class SqlDropQuark extends SqlCall {
+  public static SqlSpecialOperator operator;
+  public static String openString;
```

Comment:
same with this. don't make it static. It will make sure all it's objects have same value.